File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
import java.util.*;

class Solution {
    public int solution(int[][] sizes) {
        
        int maxW = 0;
        int maxH = 0;
        
        for(int i=0; i<sizes.length; i++){
            int max = Math.max(sizes[i][0], sizes[i][1]);
            int min = Math.min(sizes[i][0], sizes[i][1]);
            maxW = Math.max(max, maxW);
            maxH = Math.max(min, maxH);
        }
        
        return maxW*maxH;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    
    var maxW = 0;
    var maxH = 0;
    
    for(let i=0; i<sizes.length; i++){
        var max = Math.max(sizes[i][0], sizes[i][1]);
        var min = Math.min(sizes[i][0], sizes[i][1]);
        maxW = Math.max(max, maxW);
        maxH = Math.max(min, maxH);
    }
    
    return maxW*maxH;
}